src/app/admin/analytics/analytics.component.html

```diff
@@ -36,7 +36,7 @@
           <span>Nothing found</span>
         </div>
         <md-list *ngIf="users && users.admins">
-          <a *ngFor="let user of users.admins" class="online-row" [routerLink]="['/profile', user._id]">
+          <a *ngFor="let user of users.admins" class="online-row" [routerLink]="['/profile', user.username]">
```

Comment:
Was thinking about this, too. The only acceptable solution would be:
```html
[routerLink]="user.username ? ['/profile', user.username] : ['/profile/id', user._id]"
```
I think, I'll fix this.